cmd/ocsp-updater/main.go

```diff
@@ -486,7 +489,12 @@ func main() {
 		)
 
 		for _, l := range updater.loops {
-			go l.loop()
+			go func(loop *looper) {
+				err = loop.loop()
+				if err != nil {
```

Comment:
If every looper errored out, the process would continue to run but without us knowing.

Can we do this not generically, and, instead, check for this when adding the new looper to the slice?